packages/relay-kit/src/packs/safe-4337/Safe4337Pack.ts

```diff
@@ -141,6 +154,45 @@ export class Safe4337Pack extends RelayKitBasePack<{
         throw new Error('Owners and threshold are required to deploy a new Safe')
       }
 
+      const isPaymasterPresent =
+        paymasterOptions && paymasterOptions.paymasterAddress && paymasterOptions.erc20TokenAddress
+
```

Comment:
What do you think about renaming the erc20TokenAddress prop to something more meaningful as paymasterTokenAddress